phys/module_surface_driver.F

```diff
@@ -3895,6 +3895,20 @@ SUBROUTINE surface_driver(                                         &
 
 
 !------------------------------------------------------------------
+        !For WRF-CTSM simulations, we would like the land model (CTSM)
+        ! to handle inland lake points.
+        ! Here, we are changing xland to include lake points, so that
+        ! CTSM can handle it. 
+         DO j=j_start(ij),j_end(ij)
+            DO i=i_start(ij),i_end(ij)
+                IF (xland(i,j) .GT. 1.5.and.ht(i,j).gt.lake_min_elev) then
```

Comment:
@dudhia @weiwangncar @smileMchen @negin513 
Folks,
Is there not an actual WRF lake flag that can be used? I thought that we had a 2d lake flag.

Reply:
@davegill 
Thanks for your comment.  I incorporated ``lakemask(i,j).eq.1. `` in this conditional, so now it looks like this:

https://github.com/wrf-model/WRF/blob/15981e1a283516c8bd151ad77457a9e4f0c92c5c/phys/module_surface_driver.F#L3904-L3908

I am not sure how to interpret the variable ``lake_min_elev``. Does that refer to the minimum lake depth for considering a point "lake"?

I've noticed it is used in different places in module_surface_driver.F and other modules where I assumed it is used as an indicator for defining lake points.

Reply:
@negin513 @smileMchen @dudhia @weiwangncar 
Negin,
The value `lake_min_elev` has a default value set to 5 m. The idea is IF this is a water point AND the elevation of the grid cell is >= this minimum elevation, THEN this must be a lake point. This is likely an older way of identifying a lake point. 

There is a field called `lakemask`. We have an option for activating some lake physics, `sf_lake_physics`. 

Ming worked with these features.